automation/test.sh

```diff
@@ -134,24 +139,62 @@ elif [ "$TARGET" = "vagrant-release"  ]; then
 fi
 
 # Wait until kubevirt pods are running
-while [ -n "$(kubectl get pods -n kube-system --no-headers | grep -v Running)" ]; do
+let attempt=START_WAIT_TIMEOUT
+while true; do
     echo "Waiting for kubevirt pods to enter the Running state ..."
-    kubectl get pods -n kube-system --no-headers | >&2 grep -v Running || true
-    sleep 10
+    if ! kctl_out=$(kubectl get pods -n kube-system --no-headers); then
```

Comment:
It is good to have limited retries, however could you do that in a little bit more sophisticated and reusable way? Maybe with something like a backoff-retry like it is done here: https://coderwall.com/p/--eiqg/exponential-backoff-in-bash ?

Reply:
I don't think exponential backoff would be useful here, since it won't free up any usable resources effectively, since the run slave would still be held by the job.

Anyway re-factored the code to make it look a little nicer